android/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/StoreProductMapper.kt

```diff
@@ -102,28 +102,31 @@ internal fun StoreProduct.mapIntroPrice(): Map<String, Any?>? {
     }
 }
 
-private fun Period.mapPeriod(): Map<String, Any?>? {
+private fun Period.mapPeriod(forSubscriptionOption: Boolean): Map<String, Any?>? {
```

Comment:
Hmm I think it might be better to just have 2 mapping functions. I think it's different enough to warrant that?